roles/rabbitmq/tasks/main.yml

```diff
@@ -12,6 +12,14 @@
    enabled: yes
   when: manage_services|default(false)
 
+- name: Delete guest user on rabbitmq 
+  rabbitmq_user:
+     user:  '{{ rabbitmq_default_user }}' 
+     state: absent 
+  notify:
+      - restart rabbitmq
```

Comment:
It's not needed.
It doesn't hurt either.
At least it is what we said on the documentation.